### DISCUSSION TITLE
Why can vercel not deploy my pull request on github?

### DISCUSSION THREAD

**Discussion by PhilipZi** [neuefische/questions#92] [Web] [answered]
Hallo zusammen, 

bei meinem pull request ist es bei dem deployment mit Vercel zu Fehlermedlungen gekommen. Ich habe diese Fehler kontrolliert und die Ordnerstruktur die dort als Fehlerhaft angezeigt wird ist eigentlich korrekt daher verstehe ich nicht, wieso ich da weiterhin einen fehler angezeigt bekommen. 

Hier die Bilder dazu. 
![Screenshot 2022-08-22 at 09 57 06](https://user-images.githubusercontent.com/102238180/185869714-8f46c453-2a50-4695-9180-9206ec1aff94.png)

und mein PR
https://github.com/PhilipZi/capstone-project/pull/5

freue mich über jede Hilfe! Danke

**Comment by CJAkkuec** [comment, marked as answer]
"Module not found [...]" klingt oftmals danach, dass da nicht richtig importiert wird. Magst du das nochmal prüfen? :) Scheint wohl den `submitButton` in der `exerciseForm` zu betreffen. Kann auch mal ein Tippfehler o.Ä. im Export sein, etc.

**Reply by mpagels** [reply]
Immer wenn dein Deployment fehl schlägt, würde ich ein `build` Prozess `local` versuchen. Also mit `npm run build` und gucken was da raus kommt.

Das habe ich mit deinem Projekt mal gemacht und habe wie CJ geschrieben hat, den Fehler hier ausmachen können.

In `timerForm.js` hat sich in der `import` Zeile 8 ein tipper eingschlichen. `submitbutton` zu `submitButton`. Dann müsste es gehen.

**Reply by PhilipZi** [reply]
Vielen Dank für die Hilfe, ich habe npm run build durchlaufen lassen und es kam zu keiner Fehlermeldung, 
aber der Capital letter in Zeile 8 war wirklich fies :man-facepalming: 
vielen Dank!!!

**Reply by mpagels** [reply]
Okay verrück. Hier zur Info wie bei mir die Fehlermeldung aussah:
![grafik](https://user-images.githubusercontent.com/54703643/185880152-bdd8c733-01f2-4a37-9fca-3fe1316b8765.png)

**Reply by PhilipZi** [reply]
strange, bei mir was alles grün. Vielen Dank fürs gegen checken much appreciated!
